src/Components/ContentSelection/ContentSelection.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import style from './ContentSelection.module.scss'
+
+const ContentSelection = () => {
+  return (
+    <section className={style.ContentSelection}>
+      <h2 className={style.prompt}>How would you like to spend your break?</h2>
+      <div className={style.options}>
+        <div>
+          <button 
+            className={style.button} 
+            aria-label="somatic exercise"
```

Comment:
Love the attention given to accessibility with the aria labels

Reply:
+1 to what @rwilliams659 said! I love that accessibility is a priority for this project.